.github/workflows/pr-description-check.yml

```diff
@@ -10,37 +11,47 @@ permissions:
 jobs:
   check-description:
     name: Check Description
-    permissions: write-all
     runs-on: ubuntu-latest
+    outputs:
+      comment_action: ${{ steps.validate_description.outputs.comment_action }}
+      comment_content: ${{ steps.validate_description.outputs.comment_content }}
     steps:
-      - name: Check PR Description
-        uses: actions/github-script@v7
-        with:
-          script: |
-            const requiredSections = [
-              "## Test plan",
-              "## Documentation update"
-            ];
-            
-            const prBody = context.payload.pull_request.body || "";
+      - name: Validate Description
+        id: validate_description
+        run: |
+          required_sections=("## Description" "## Test plan" "## Documentation update")
```

Comment:
would be nice if this was broken out elsewhere for visibility, like an env var

https://stackoverflow.com/a/73274812/9806233

Reply:
Split into env